addons/bus/bus.py

```diff
@@ -114,7 +114,7 @@ def loop(self):
         with openerp.sql_db.db_connect('postgres').cursor() as cr:
             conn = cr._cnx
             cr.execute("listen imbus")
-            cr.commit();
+            cr.commit()
```

Comment:
Please don't touch lines that are unrelated to your PR, to avoid noise in PRs and in history.